### ISSUE TITLE
Effect Layer has no influence on the final visual

### ISSUE THREAD

**Issue by jeffreyschoch** [HuvaaKoodia/2DMetaballs#1]
Just tried out this project and found that using the "Effect Layer" influences what the Effect Camera captures, but does not influence what is processed for the Metaball image effect.  All sprites will be detected in the AlphaTest unless rendered by a camera with a higher depth than the effect camera.

**Comment by HuvaaKoodia** [comment]
I looked around and apparently there isn't an easy, official way to apply image effects to individual cameras or individual layers. Everything rendered with and before the image effect camera will be affected by the image effect script.

Fortunately I found a solution for drawing sprites before and after the image effect by using multiple cameras with different clear flags. Objects using the Background layer will be drawn behind the metaballs and other objects will be drawn on top of the metaballs. 

A three camera setup is not optimal, but should suffice for a 2D project.
